yumljira/apps/projects/serializers.py

```diff
@@ -37,3 +37,21 @@ class Meta:
         model = Project
         fields = ('name', 'created_by', 'tasks')
 
+
+class TimeLogSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = TimeLog
+        fields = ('user', 'task', 'date', 'time_logged')
+
+    def validate_task(self, task):
+        if not task:
+            raise ValidationError({'task': [_('You have to assign issue first.')]})
+
+        return task
+
+    def validate_time_logged(self, time_logged):
+        if time_logged <= 0:
+            raise ValidationError({'time_logged': [_('You cannot log 0 minutes.')]})
```

Comment:
Imho here error message should be more like "You need log more than 0 minutes". I thinking about the situation when user insert in some case value like (-10).